packages/server/src/graphql/member/resolver.ts

```diff
@@ -0,0 +1,62 @@
+import { Arg, Ctx, Mutation, Query, Resolver } from "type-graphql";
+import { Context } from "../../context";
+import { MemberInput } from "./inputs/memberInput";
+import { UpdateMemberResult } from "./results/UpdateMemberResult";
+import { DeleteMemberResult } from "./results/DeleteMemberResult";
+import { validate } from "class-validator";
+import { FieldsValidationError } from "../common/errors/FieldsValidationError";
+import { NonExistingMemberError } from "./errors/NonExistingMemberError";
+import { TenantUser } from "../../generated/typegraphql-prisma/models/TenantUser";
+import { Role } from "../../generated/typegraphql-prisma/enums/Role";
+
+@Resolver()
+class MemberResolver {
+  @Query(() => [TenantUser], { nullable: true })
+  async getMembers(@Ctx() ctx: Context): Promise<TenantUser[] | null> {
+    return await ctx.prisma.tenantUser.findMany({
+      include: {
+        User: true,
+        Tenant: true,
+      },
+    });
+  }
+
+  @Mutation(() => DeleteMemberResult)
+  async deleteMember(
+    @Arg("input", () => MemberInput) input: MemberInput,
+    @Ctx() ctx: Context
+  ): Promise<typeof DeleteMemberResult> {
```

Comment:
Arguments:
- MemberId

Return type: _you can choose on of_
1. bool: true if deleted / false otherwise
1. TenantUser | Error: deleted member data or error

We don't need the tenantId as it will be handled by the RLS already